packages/dm-core-plugins/src/generic-list/GenericListPlugin.tsx

```diff
@@ -161,170 +147,144 @@ export const GenericListPlugin = (
-          <tr style={{ borderBottom: '1px solid rgba(224, 224, 224, 1)' }}>
-            <th></th>
-            {internalConfig.headers.map((attribute: string) => (
-              <th key={attribute}>
-                <Typography group="table" variant={'cell_header'}>
-                  {attribute}
-                </Typography>
-              </th>
-            ))}
-            <th>
-              <Typography group="table" variant={'cell_header'}>
-                Controls
-              </Typography>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          {Object.entries(paginatedItems(items, currentPage, itemsPerPage)).map(
-            ([key, item]) => (
-              <React.Fragment key={key}>
-                <TableRow>
-                  <td style={{ textAlign: 'left' }}>
-                    <Button
-                      variant="ghost_icon"
-                      onClick={() =>
-                        setItemsExpanded({
-                          ...itemsExpanded,
-                          [key]: !itemsExpanded[key],
-                        })
-                      }
-                    >
-                      <Icon
-                        data={itemsExpanded[key] ? chevron_down : chevron_right}
-                        title="Expand"
-                      />
-                    </Button>
-                  </td>
-                  {internalConfig.headers.map((attribute: string) => {
-                    if (item.data && attribute in item.data) {
-                      if (typeof item.data[attribute] === 'object')
-                        throw new Error(
-                          `Objects can not be displayed in table header. Attribute '${attribute}' is not a primitive type.`
-                        )
-                      return (
-                        <TableData key={attribute}>
-                          <Typography group="table" variant={'cell_text'}>
-                            {item?.data[attribute]}
-                          </Typography>
-                        </TableData>
+    <Stack>
+      {paginatedRows.map((item, index) => (
+        <React.Fragment key={item?.key}>
+          <Stack
+            direction="row"
+            spacing={1}
+            justifyContent="space-between"
+            alignItems="center"
+            style={{
+              padding: '0.25rem 0.5rem',
+              borderBottom: '1px solid #ccc',
+            }}
+          >
+            <Stack direction="row" spacing={1} alignItems="center">
+              <Tooltip
+                title={
+                  internalConfig.openInline
+                    ? item.expanded
+                      ? 'Minimize'
+                      : 'Expand'
+                    : 'Open in new tab'
+                }
+              >
+                <Button
+                  variant="ghost_icon"
+                  color="secondary"
+                  onClick={
+                    internalConfig.openInline
```

Comment:
I think there is a bug related to this button click

when clicking the + icon several times for the same case, several tabs will open. 
I guess the wanted functionality is to jump to the already opened tab.

![image](https://github.com/equinor/dm-core-packages/assets/69512295/5673e2d2-edd9-4a49-9d58-1754e78b3b60)

Reply:
I mentioned this very briefly yesterday I think, but I also feel like this is a separate PR 'cause it's more related to the attribute-selector and how we add views 👍🏻 I have plans to fix it though